util/doxygen/header.html

```diff
@@ -0,0 +1,56 @@
+<!-- HTML header for doxygen 1.8.17-->
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
+<meta http-equiv="X-UA-Compatible" content="IE=9"/>
+<meta name="generator" content="Doxygen $doxygenversion"/>
+<meta name="viewport" content="width=device-width, initial-scale=1"/>
+<!--BEGIN PROJECT_NAME--><title>$projectname: $title</title><!--END PROJECT_NAME-->
+<!--BEGIN !PROJECT_NAME--><title>$title</title><!--END !PROJECT_NAME-->
+<link href="$relpath^tabs.css" rel="stylesheet" type="text/css"/>
+<script type="text/javascript" src="$relpath^jquery.js"></script>
```

Comment:
Can we pull this from a CDN? It looks like this and `dynsections.js` might not be loading right now given they're expected to be there by default.

Reply:
Oh I had hoped to turn off a lot of the dynamic stuff. I'll take another look but really I want to do major changes to the layouts.

Reply:
So `jquery.js` and `dynsections.js` both get copied into the doxygen path as part of generating the documentation, but I have replaced `jquery.js` with two CDN links (removing it entirely also seems to give no issues, given i switched off all the "dynamic" features).